lib/backend/etcdv3/etcdv3_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 var _ = Describe("RulesAPIToBackend", func() {
-	It("should raise an error if specified certs don't exist", func() {
+	It("should raise an error if specified certs files don't exist", func() {
```

Comment:
Please add a test for the "success" case.  I.e. valid key, cert and CA cert.

Reply:
I have added success test case.